### PR TITLE
Commission Plaques

### PR DESCRIPTION
**Pull request by EOBGames**
## About The Pull Request
Adds commission plaques- these commemorate the day a station was added to the game (adjusted for SS13 time) and the day it was removed (in the case of removed stations). If I got any of the dates wrong, please feel free to tell me.

They're added to every in rotation station, as well as the Derelict and Runtime Station.

Many thanks to Fikou for providing the git hashes for each commit to add/remove a station:

946ec1fec869eb59d6a84e32c90613af734fcd0e - Runtime ADD
10d846022a91f79642b9d578ac4103b48bf7b475 - MultiZ Debug ADD

8af93a638b8a25a7327f99c401297ad039a307a3 - IceBox ADD
b02c4595053a9e55b3c100952732753812865cee - Meta ADD
f0bcbe37683c6dbb083a675b734c6b0e62046d7d - Delta ADD
3451a1f9199321d8347bb676326d3138b187bfa3 - Kilo ADD
5f3e12178e1fbe75ae4b1ff12e256f61b5bdddd5 - Tram ADD

169ab09f7b52254ee505e54cdea681fab287647b - Asteroid ADD
5d2c235e7caee1e488979cd7b0eb299568a06669 - Asteroid DEL
7dfef9b58171c118e060055af409673588b8e40b - Birdboat ADD
ca458248c007b2a2a07ec37e2f75ae24535d7e74 - Birdboat DEL
6438f2e6124eb487616a9b5c845a8c32455bf0c9 - Box ADD
eca1ebad92408854774e54b716deee3016b8a2f2 - Box DEL
b26c78c9e3d98a29366fe7c21b7d7dc576ef8891 - Pubby ADD
22ca230db02d2201c8c6e53b00db1021f1fd0fcb - Pubby DEL
48da27feb804527a602205a5832d65dd8fcd249e - Cere ADD
cc33efbf8655519dd06d7a526559002887c1d120 - Cere DEL
34c539fa2d83817f89d80a2975f179647655713a - Disc ADD
7ec92316386603abc17e8e6406e44e00469e50ab - Disc DEL
a5f5aee5a2ccd64c38591a4819df26542697f136 - Donut ADD
a4fe34bf4ffdb05ebcf517e59731f2fa34d8ad32 - Donut DEL
1c34cff202e5d191d84528b3e04c1fff05321588 - Dream ADD
c9f66573c58d9f263e55365327e67f169ed6b61e - Dream DEL
46f64266cfb8b40e35faa8a4d9a2d3aeec689943 - Efficiency ADD
e1efad2dfd3c84acfa4db9484139056bcf1e79e5 - Efficiency DEL
7a76e9456b782e6626bf81e27a912d8232c76b18 - Mini ADD
bf3203bc6523f63922b47fce72a774955092c3ff - Mini DEL
4b6a048f5668a6b7883407a80da203154bea08b0 - Omega ADD
a7fa8933ab014c9fa7a07338f2c9ed8b1de2466e - Omega DEL
bbd6db9ce2d6341892b89a620593fc8877f5a817 - Uterus ADD
72d72f7ce522c2d2ad4863f44ee9f5054413c489 - Uterus DEL

## Why It's Good For The Game
It's a neat nod to the maps' OOC history, as well as simply being some nice flavour.

## Changelog
:cl: Inept, Fikou
add: Nanotrasen has seen it fit to officially commemorate the commission date of its various stations- look for the commission plaques on each station today!
/:cl:

### CODE REVIEW

File: code/game/objects/structures/plaques/static_plaques.dm [hunk 1]
```diff
@@ -19,6 +19,72 @@
...
+
+/obj/structure/plaque/static_plaque/golden/commission
+	name = "Commission Plaque"
+	desc = "Spinward Sector Station SS-13\n'Runtime' Class Outpost\nCommissioned 11/03/2556\n'Dedicated to the Pioneers'"
+	icon_state = "commission_nt"
+
+//Current stations
+
+/obj/structure/plaque/static_plaque/golden/commission/icebox
+	desc = "Spinward Sector Station SS-13\n'Box' Class Outpost (Revivion 2.2: 'Icebox')\nCommissioned 05/22/2560\n'Cold Reliable'"
```
Comment (coiax) [inline code/game/objects/structures/plaques/static_plaques.dm:32]:
```suggestion
	desc = "Spinward Sector Station SS-13\n'Box' Class Outpost (Revision 2.2: 'Icebox')\nCommissioned 05/22/2560\n'Cold Reliable'"
```

File: code/game/objects/structures/plaques/static_plaques.dm [hunk 2]
```diff
@@ -19,6 +19,72 @@
 /obj/structure/plaque/static_plaque/golden/captain
 	name = "The Most Robust Captain Award for Robustness"
 
+//Commission plaques, to give a little backstory to the stations. Commission dates are date of merge (or best approximation, in the case of Meta) + 540 years to convert to SS13 dates.
+
+/obj/structure/plaque/static_plaque/golden/commission
+	name = "Commission Plaque"
+	desc = "Spinward Sector Station SS-13\n'Runtime' Class Outpost\nCommissioned 11/03/2556\n'Dedicated to the Pioneers'"
```
Comment (coiax) [inline code/game/objects/structures/plaques/static_plaques.dm:26]: Is it out of scope to leave a comment of a date and git commit hash (if you can get one), so you can explain where these dates came from?

**Review by coiax — COMMENTED** [review]
Why no ISO 8601 dates 😭

**Comment by Fikou** [comment]
unamericanize the dates/.....

File: code/game/objects/structures/plaques/static_plaques.dm (hunk 2 above)
Comment (EOBGames) [reply code/game/objects/structures/plaques/static_plaques.dm:26]: yeah, I can add those, at least where I can get them

File: code/game/objects/structures/plaques/static_plaques.dm [hunk 3]
```diff
@@ -19,6 +19,75 @@
...
+/obj/structure/plaque/static_plaque/golden/commission/kilo
+	desc = "Spinward Sector Station SS-13\n'Kilo' Class Outpost\nCommissioned 11/13/2559\n'Forever Different'"
+
+/obj/structure/plaque/static_plaque/golden/commission/tram
+	desc = "Spinward Sector Station SS-13\n'Tram' Class Outpost\nCommissioned 03/11/2561\n'Making Moves'"
+
+//Removed stations
+
+/obj/structure/plaque/static_plaque/golden/commission/asteroid
+	desc = "Spinward Sector Station SS-12\n'Asteroid' Class Outpost\nCommissioned 10/13/2555\nDecommissioned 06/19/2556\n'A Meteoric Success'"
```
Comment (Fikou) [inline code/game/objects/structures/plaques/static_plaques.dm:49]: it was merged on 17th

Comment (Fikou) [reply code/game/objects/structures/plaques/static_plaques.dm:49]: and removed on 4th april 2016

File: code/game/objects/structures/plaques/static_plaques.dm [hunk 4]
```diff
@@ -19,6 +19,75 @@
...
+
+/obj/structure/plaque/static_plaque/golden/commission/birdboat
+	desc = "Spinward Sector Station SS-03\n'Birdboat' Class Outpost\nCommissioned 09/17/2555\nDecommissioned 02/09/2557\n'Rocking the Boat'"
```
Comment (Fikou) [inline code/game/objects/structures/plaques/static_plaques.dm:52]: birdboat removal was 5th of february

File: code/game/objects/structures/plaques/static_plaques.dm [hunk 5]
```diff
@@ -19,6 +19,75 @@
...
+	desc = "Spinward Sector Station SS-02\n'Box' Class Outpost\nCommissioned 11/15/2550\nDecommissioned 07/06/2560\n'Old Faithful'"
+
+/obj/structure/plaque/static_plaque/golden/commission/pubby
+	desc = "Spinward Sector Station SS-06\n'Pubby' Class Outpost\nCommissioned 10/19/2556\nDecommissioned 12/25/2560\n'No Law But Ours'"
+
+/obj/structure/plaque/static_plaque/golden/commission/cere
+	desc = "Spinward Sector Station SS-10\n'Cere' Class Outpost\nCommissioned 03/29/2557\nDecommissioned 08/26/2557\n'Take a Hike'"
+
+/obj/structure/plaque/static_plaque/golden/commission/disc
+	desc = "Spinward Sector Station SS-05\n'Disc' Class Outpost\nCommissioned 09/21/2555\nDecommissioned 01/31/2556\n'Sleep Tight'"
```
Comment (Fikou) [inline code/game/objects/structures/plaques/static_plaques.dm:64]: removal was 31, not 30

File: code/game/objects/structures/plaques/static_plaques.dm [hunk 6]
```diff
@@ -19,6 +19,75 @@
...
+
+/obj/structure/plaque/static_plaque/golden/commission/donut
+	desc = "Spinward Sector Station SS-11\n'Donut' Class Outpost\nCommissioned 12/16/2558\nDecommissioned 04/28/2560\n'Hail the Lord'"
+
+/obj/structure/plaque/static_plaque/golden/commission/dream
+	desc = "Spinward Sector Station SS-04\n'Dream' Class Outpost\nCommissioned 08/11/2555\nDecommissioned 12/22/2556\n'Aiming High'"
```
Comment (Fikou) [inline code/game/objects/structures/plaques/static_plaques.dm:70]: removal on 20th

Comment (Fikou) [reply code/game/objects/structures/plaques/static_plaques.dm:70]: added on 4th

File: code/game/objects/structures/plaques/static_plaques.dm [hunk 7]
```diff
@@ -19,6 +19,75 @@
...
+	desc = "Spinward Sector Station SS-13\n'Runtime' Class Outpost\nCommissioned 11/03/2556\n'Dedicated to the Pioneers'"
+	icon_state = "commission_nt"
+
+//Current stations
+
+/obj/structure/plaque/static_plaque/golden/commission/icebox
+	desc = "Spinward Sector Station SS-13\n'Box' Class Outpost (Revivion 2.2: 'Icebox')\nCommissioned 05/22/2560\n'Cold Reliable'"
+
+/obj/structure/plaque/static_plaque/golden/commission/meta
+	desc = "Spinward Sector Station SS-13\n'Meta' Class Outpost\nCommissioned 08/11/2553\n'Theseus' Station'"
```
Comment (Fikou) [inline code/game/objects/structures/plaques/static_plaques.dm:35]: closest accurate date we have is 11 march

File: code/game/objects/structures/plaques/static_plaques.dm [hunk 8]
```diff
@@ -19,6 +19,75 @@
...
+	desc = "Spinward Sector Station SS-11\n'Donut' Class Outpost\nCommissioned 12/16/2558\nDecommissioned 04/28/2560\n'Hail the Lord'"
+
+/obj/structure/plaque/static_plaque/golden/commission/dream
+	desc = "Spinward Sector Station SS-04\n'Dream' Class Outpost\nCommissioned 08/11/2555\nDecommissioned 12/22/2556\n'Aiming High'"
+
+/obj/structure/plaque/static_plaque/golden/commission/efficiency
+	desc = "Spinward Sector Station SS-07\n'Efficiency' Class Outpost\nCommissioned 01/28/2556\nDecommissioned 12/20/2556\n'Work Smarter, Not Harder'"
+
+/obj/structure/plaque/static_plaque/golden/commission/mini
+	desc = "Spinward Sector Station SS-08\n'Mini' Class Outpost\nCommissioned 01/29/2554\nDecommissioned 12/27/2556\n'The Littlest Station'"
```
Comment (Fikou) [inline code/game/objects/structures/plaques/static_plaques.dm:76]: removal 25th

File: code/game/objects/structures/plaques/static_plaques.dm [hunk 9]
```diff
@@ -19,6 +19,75 @@
...
+	desc = "Spinward Sector Station SS-05\n'Disc' Class Outpost\nCommissioned 09/21/2555\nDecommissioned 01/31/2556\n'Sleep Tight'"
+
+/obj/structure/plaque/static_plaque/golden/commission/donut
+	desc = "Spinward Sector Station SS-11\n'Donut' Class Outpost\nCommissioned 12/16/2558\nDecommissioned 04/28/2560\n'Hail the Lord'"
+
+/obj/structure/plaque/static_plaque/golden/commission/dream
+	desc = "Spinward Sector Station SS-04\n'Dream' Class Outpost\nCommissioned 08/11/2555\nDecommissioned 12/22/2556\n'Aiming High'"
+
+/obj/structure/plaque/static_plaque/golden/commission/efficiency
+	desc = "Spinward Sector Station SS-07\n'Efficiency' Class Outpost\nCommissioned 01/28/2556\nDecommissioned 12/20/2556\n'Work Smarter, Not Harder'"
```
Comment (Fikou) [inline code/game/objects/structures/plaques/static_plaques.dm:73]: removal 19th

File: code/game/objects/structures/plaques/static_plaques.dm (hunk 8 above)
Comment (EOBGames) [reply code/game/objects/structures/plaques/static_plaques.dm:76]: was removed 27th, PR was made 25th

Comment (Fikou) [reply code/game/objects/structures/plaques/static_plaques.dm:76]: commits from earlier stations get really confusing, nowadays i think it lists the commit as the one where it was merged, not the authors one

**Comment by EOBGames** [comment]
Dates have been successfully unyankeed, the freedom levels in this PR are down 95% as a result

File: code/game/objects/structures/plaques/static_plaques.dm [hunk 10]
```diff
@@ -19,6 +19,96 @@
...
+
+// Asteroidstation: added Oct 17, 2015 (169ab09f7b52254ee505e54cdea681fab287647b), removed Jun 19, 2016 (#18661)- 8 months, 2 days
+/obj/structure/plaque/static_plaque/golden/commission/asteroid
+	desc = "Spinward Sector Station SS-12\n'Asteroid' Class Outpost\nCommissioned 13/10/2555\nDecommissioned 19/06/2556\n'A Meteoric Success'"
+
+// Birdboat Station: added Sep 17, 2015 (#11829), removed Feb 09, 2017 (#23754)- 1 year, 4 months, 23 days
+/obj/structure/plaque/static_plaque/golden/commission/birdboat
+	desc = "Spinward Sector Station SS-03\n'Birdboat' Class Outpost\nCommissioned 17/09/2555\nDecommissioned 09/02/2557\n'Rocking the Boat'"
+
+// Boxstation: added Nov 15, 2010 (pre-git), removed Jul 06, 2020 (#52017)- 9 years, 7 months, 21 days
```
Comment (Fikou) [inline code/game/objects/structures/plaques/static_plaques.dm:64]: we have an exact commit for this, cant we link the hash?

**Comment by maxymax13** [comment]
american dates be like ah yes, it is the seventh of twentynineber

**Comment by Twaticus** [comment]
these dates are impossible to read bro non americans really live like this?

**Comment by maxymax13** [comment]
> 
> 
> these dates are impossible to read bro non americans really live like this?

americans be like hmm yes it is the julyth of four of 2021

**Review by Twaticus — APPROVED** [review]

**Comment by Twaticus** [comment]
its may 10th

File: code/game/objects/structures/plaques/static_plaques.dm [hunk 11]
```diff
@@ -19,6 +19,96 @@
...
+	desc = "Spinward Sector Station SS-09\n'Omega' Class Outpost\nCommissioned 27/12/2556\nDecommissioned 20/09/2558\n'Tiny Take Two'"
+
+// Uterusstation: added Sep 03, 2011 (bbd6db9ce2d6341892b89a620593fc8877f5a817), removed Jun 21, 2012 (72d72f7ce522c2d2ad4863f44ee9f5054413c489)- 9 months, 18 days
+/obj/structure/plaque/static_plaque/golden/commission/uterus
+	desc = "Spinward Sector Station SS-01\n'Uterus' Class Outpost\nCommissioned 03/09/2551\nDecommissioned 21/06/2552\n'Humanity's Vanguard'"
+
+// Other Stations
+
+// Space Station 13, Developer Class Outpost, Station Commissioned 30.12.2322, For the Glory of the Workers of the Third Soviet Union
+// The date for this station is exactly 4 centuries after the initial foundation of the (1st) Soviet Union. I have very little interest in tracking down the actual date of addition of Originalstation.
```
Comment (CRITAWAKETS) [inline code/game/objects/structures/plaques/static_plaques.dm:107]: PS if interested: the build from 2003 we have has an early version of originalstation, so its very probable its as old as SS13 itself

File: code/game/objects/structures/plaques/static_plaques.dm [hunk 12]
```diff
@@ -19,6 +19,96 @@
...
+
+//Removed stations
+
+// Asteroidstation: added Oct 17, 2015 (169ab09f7b52254ee505e54cdea681fab287647b), removed Jun 19, 2016 (#18661)- 8 months, 2 days
+/obj/structure/plaque/static_plaque/golden/commission/asteroid
+	desc = "Spinward Sector Station SS-12\n'Asteroid' Class Outpost\nCommissioned 13/10/2555\nDecommissioned 19/06/2556\n'A Meteoric Success'"
+
+// Birdboat Station: added Sep 17, 2015 (#11829), removed Feb 09, 2017 (#23754)- 1 year, 4 months, 23 days
+/obj/structure/plaque/static_plaque/golden/commission/birdboat
+	desc = "Spinward Sector Station SS-03\n'Birdboat' Class Outpost\nCommissioned 17/09/2555\nDecommissioned 09/02/2557\n'Rocking the Boat'"
```
Comment (coiax) [inline code/game/objects/structures/plaques/static_plaques.dm:62]:
```suggestion
	desc = "Spinward Sector Station SS-03\n'Birdboat' Class Outpost\nCommissioned 2555-09-17\nDecommissioned 2557-02-09\n'Rocking the Boat'"
```

ISO dates are YYYY-MM-DD. Same for the others.

**Review by coiax — CHANGES_REQUESTED** [review]

**Pull request merged**